src/Sarif/VersionConstants.cs

```diff
@@ -5,8 +5,8 @@ namespace Microsoft.CodeAnalysis.Sarif
     public static class VersionConstants                                       
     {                                                                          
         public const string Prerelease = "-beta";                       
-        public const string AssemblyVersion = "1.5.21";       
-        public const string FileVersion = "1.5.21" + ".0";    
+        public const string AssemblyVersion = "1.5.22";       
+        public const string FileVersion = "1.5.22" + ".0";    
```

Comment:
No idea. I didn't touch it. Is it possible you checked in a change to SetCurrentVersion.cmd but didn't check in the corresponding changes to the generated files?

---
In reply to: [62771652](https://github.com/Microsoft/sarif-sdk/pull/257#discussion_r62771652) [](ancestors = 62771652)